### PR TITLE
docs(alert): add customization playground

### PR DESCRIPTION
**Pull request by amandaejohnston**
Adds the Customization playground and removes a couple of dead anchor links from the docs.

### CODE REVIEW

File: static/usage/alert/customization/angular/angular_ts.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+```ts
+import { Component } from '@angular/core';
+import { AlertController } from '@ionic/angular';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: 'app.component.html',
+})
+export class AppComponent {
+  constructor(public alertController: AlertController) {}
```
Comment (sean-perkins) [inline static/usage/alert/customization/angular/angular_ts.md:10]:
```suggestion
  constructor(private alertController: AlertController) {}
```

File: docs/api/alert.md [hunk 2]
```diff
@@ -154,9 +154,15 @@ Any of the defined [CSS Custom Properties](#css-custom-properties) can be used t
 ```
 
 :::note
- If you are building an Ionic Angular app, the styles need to be added to a global stylesheet file. Read [Style Placement](#style-placement) in the Angular section below for more information.
+ If you are building an Ionic Angular app, the styles need to be added to a global stylesheet file.
 :::
 
+### Example
```
Comment (sean-perkins) [inline docs/api/alert.md:160]: If this example is focused around styling, can we update this heading to be "Theming" or "Styling"?

**Review by liamdebeasi — CHANGES_REQUESTED** [review]
Can we make this playground look a bit more "real world"? For example, Chakra UI has a nice sample here: https://chakra-ui.com/docs/components/alert-dialog/usage.

Optional: It might be valuable to keep the theming consistent. For example, Tailwind's samples all have consistent themes with coordinated colors: https://tailwindcss.com/ (in the "Build whatever you want, seriously." section)

**Comment by amandaejohnston** [comment]
Good point! Changed to use the `success` colors instead 👍

**Comment by sean-perkins** [comment]
Screen Shot 2022-06-30 at 12 50 38 PM

The iOS preview could still use some refinement. The cancel button is clipping and the confirm button border radius does not match consistently.

**Comment by amandaejohnston** [comment]
Ah, that'd be from the buttons wrapping; your screen size is likely a bit smaller than mine. I'll just make it a simple yes/no to avoid that.

**Review by sean-perkins — APPROVED** [review]
Alert looks great now 👍

**Pull request merged**